source/api/image/3.0/index.md

```diff
@@ -136,25 +136,27 @@ The API places no restrictions on the form of the identifiers that a server may
 
 ##  4. Image Requests
 
-All parameters described below are required for compliant construction of a IIIF Image API URI. The sequence of parameters in the URI _MUST_ be in the order described below. The order of the parameters is also intended as a mnemonic for the order of the operations by which the service should manipulate the image content. Thus, the requested image content is first extracted as a region of the complete image, then scaled to the requested size, mirrored and/or rotated, and finally transformed into the color quality and format. This resulting image content is returned as the representation for the URI. 
+This specification makes no assumptions about the form of the underlying image content. The term **full image** is used to refer to the entire area of the underlying image content, with the pixel dimensions given in the [Image Information][image-information], and which is imagined as the starting point for image requests.
```

Comment:
> the form of the underlying image content.

Strengthen this with examples, e.g. the content might be derived from a source image, but may also be dynamically generated ...

Reply:
"...imagined as..." -> considered as?

Reply:
Happy to add examples, would prefer to keep "imagined" over "considered"